spec/index.bs

```diff
@@ -2282,6 +2282,29 @@ dictionary GPUImageBitmapCopyView {
 
   * {{GPUImageBitmapCopyView/origin}}: If unspecified, defaults to `[0, 0]`.
 
+### {{GPUCommandEncoder/copyBufferToBuffer()|GPUCommandEncoder.copyBufferToBuffer(GPUBuffer, GPUSize64, GPUBuffer, GPUSize64, GPUSize64)}} ### {#GPUCommandEncoder-copyBufferToBuffer}
+
+<div algorithm="GPUCommandEncoder.copyBufferToBuffer(GPUBuffer, GPUSize64, GPUBuffer, GPUSize64, GPUSize64)">
+
+The <dfn method for="GPUCommandEncoder">copyBufferToBuffer(|source|, |sourceOffset|, |destination|, |destinationOffset|, |size|)</dfn> method is used to perform {{GPUBuffer}} to {{GPUBuffer}} copies.
+
+1. Ensure {{GPUCommandEncoder/copyBufferToBuffer()}} is called before {{GPUCommandEncoder/finish()}}.
+1. Ensure {{GPUCommandEncoder/copyBufferToBuffer()}} is called outside of any {{GPURenderPassEncoder}}s.
+1. Ensure {{GPUCommandEncoder/copyBufferToBuffer()}} is called outside of any {{GPUComputePassEncoder}}s.
+1. Ensure |source| is a valid {{GPUBuffer}}.
+1. Ensure |destination| is a valid {{GPUBuffer}}.
+1. Ensure the {{GPUBuffer/[[usage]]}} of |source| contains {{GPUBufferUsage/COPY_SRC}}.
+1. Ensure the {{GPUBuffer/[[usage]]}} of |destination| contains {{GPUBufferUsage/COPY_DST}}.
+1. Ensure |size| is a multiple of 4.
+1. Ensure |sourceOffset| is a multiple of 4.
+1. Ensure |destinationOffset| is a multiple of 4.
+1. Ensure (|sourceOffset| + |size|) does not overflow.
```

Comment:
nit: does not overflow a {{GPUBufferSize}}. I'm not sure how we should handle overflows in the spec though, we should ask spec editors.

Reply:
I'm not sure yet either, we'll have to figure this one out eventually. Can you add a TODO for this as well?

Reply:
Done